docs/builtins.md

```diff
@@ -606,22 +652,28 @@ print arr[i] // out of bounds
 print "This line will not be executed"
 ```
 
-This code will cause a fatal error because the index 5 is out of bounds
+This code will cause a panic because the index 5 is out of bounds
 for the array `arr`. The program will exit with the error message
 
 ```
-line 3: index out of bounds: 5
+line 3: panic: index out of bounds: 5
 ```
 
-### Non-fatal Errors
+If you want your own code to panic, you can use the built-in `panic`
+function. The panic exits your program immediately. It is typically
+used when something unexpected happens, such as invalid user input or a
```

Comment:
I would have thought that invalid user input is something you would typically not panic on. At least I don't think we should be encouraging people to panic on bad input.

Reply:
fair enough... I've just removed the `such as...` part
I've removed the user input comment in the sample above too. I can't really think of a good example at the moment.